app/views/projects/blob/_actions.html.haml

```diff
@@ -17,6 +17,10 @@
         tree_join(@commit.sha, @path)), class: 'btn btn-sm'
 
 - if allowed_tree_edit?
-  = button_tag class: 'remove-blob btn btn-sm btn-remove',
+  = button_tag class: 'btn btn-small btn-primary btn-xs',
+      'data-toggle' => 'modal', 'data-target' => '#modal-replace-blob' do
+    Replace	 
+  = |
```

Comment:
Could this be done with a button group?

http://getbootstrap.com/components/#btn-groups